.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '6.10.2'
+- '10.15.3'
```

Comment:
Sorry, I know I'm being an extreme nitpicker here, but this is the only place in any `.yml` file where we use single quotes, everywhere else is double. Can we be consistent?

Reply:
What if I remove all the quotes? It won't matter and it's again for keeping consistency

Reply:
Done!